projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_print.php

```diff
@@ -160,228 +173,237 @@ function formatted_page($doctype='xhtml1.1', $externalcss=true, $title='', $lang
-				if ($this->parser->get_cfg('sort_properties'))
-					ksort($vali);
-				$this->parser->_add_token(SEL_START, $selector, true);
+			foreach ( $val as $selector => $vali ) {
+				if ( $this->parser->get_cfg( 'sort_properties' ) ) {
+					ksort( $vali );
+				}
+				$this->parser->_add_token( SEL_START, $selector, true );
 
-				foreach ($vali as $property => $valj) {
-					$this->parser->_add_token(PROPERTY, $property, true);
-					$this->parser->_add_token(VALUE, $valj, true);
+				foreach ( $vali as $property => $valj ) {
+					$this->parser->_add_token( PROPERTY, $property, true );
+					$this->parser->_add_token( VALUE, $valj, true );
 				}
 
-				$this->parser->_add_token(SEL_END, $selector, true);
+				$this->parser->_add_token( SEL_END, $selector, true );
 			}
 
 			if ( (int) $medium < DEFAULT_AT ) {
-				$this->parser->_add_token(AT_END, $medium, true);
-			}
-			elseif ($default_media) {
-				$this->parser->_add_token(AT_END, $default_media, true);
+				$this->parser->_add_token( AT_END, $medium, true );
+			} elseif ( $default_media ) {
+				$this->parser->_add_token( AT_END, $default_media, true );
 			}
 		}
 	}
 
 	/**
 	 * Same as htmlspecialchars, only that chars are not replaced if $plain !== true. This makes  print_code() cleaner.
-	 * @param string $string
-	 * @param bool $plain
+	 *
+	 * @param string $string - the string we're converting.
+	 * @param bool   $plain - plain text or not.
 	 * @return string
 	 * @see csstidy_print::_print()
 	 * @access private
 	 * @version 1.0
 	 */
-	function _htmlsp($string, $plain) {
-		if (!$plain) {
-			return htmlspecialchars($string, ENT_QUOTES, 'utf-8');
+	public function htmlsp( $string, $plain ) {
+		if ( ! $plain ) {
+			return htmlspecialchars( $string, ENT_QUOTES, 'utf-8' );
 		}
 		return $string;
 	}
 
 	/**
 	 * Get compression ratio
+	 *
 	 * @access public
 	 * @return float
 	 * @version 1.2
 	 */
-	function get_ratio() {
-		if (!$this->output_css_plain) {
+	public function get_ratio() {
+		if ( ! $this->output_css_plain ) {
 			$this->formatted();
 		}
-		return round((strlen($this->input_css) - strlen($this->output_css_plain)) / strlen($this->input_css), 3) * 100;
+		return round( ( strlen( $this->input_css ) - strlen( $this->output_css_plain ) ) / strlen( $this->input_css ), 3 ) * 100;
 	}
 
 	/**
 	 * Get difference between the old and new code in bytes and prints the code if necessary.
+	 *
 	 * @access public
 	 * @return string
 	 * @version 1.1
 	 */
-	function get_diff() {
-		if (!$this->output_css_plain) {
+	public function get_diff() {
+		if ( ! $this->output_css_plain ) {
 			$this->formatted();
 		}
 
-		$diff = strlen($this->output_css_plain) - strlen($this->input_css);
+		$diff = strlen( $this->output_css_plain ) - strlen( $this->input_css );
 
-		if ($diff > 0) {
+		if ( $diff > 0 ) {
 			return '+' . $diff;
-		} elseif ($diff == 0) {
+		} elseif ( $diff == 0 ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
I see no way `$diff` won't be an integer. Even if `strlen()` did somehow return a non-integer, the subtraction would make it one.